### PR TITLE
Some issue fixes

### PR DESCRIPTION
**Pull request by MertUnverdi**

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/database/Island.java [hunk 1]
```diff
@@ -115,6 +106,15 @@ public void setHome(@NotNull Location location) {
         this.home = homeLocation.getX() + "," + homeLocation.getY() + "," + homeLocation.getZ() + "," + homeLocation.getPitch() + "," + homeLocation.getYaw();
     }
 
+    /**
+     * Milliseconds of date this island was created
+     *
+     * @return A LocalDateTime of this island was created
+     */
+    public LocalDateTime getCreateTime() {
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/database/Island.java:114]: I just wrapped the setHome and getHome methods at here

File: src/main/java/com/iridium/iridiumskyblock/configs/Messages.java [hunk 2]
```diff
@@ -23,6 +23,7 @@
     public String invitedPlayer = "%prefix% &7You have invited %player% to join your island";
     public String userInvitedPlayer = "%prefix% &7%inviter% has invited %player% to join your island";
     public String youHaveBeenInvited = "%prefix% &7%inviter% has invited you to join their island";
+    public String alreadyInYourIsland = "%prefix% &7This player is already in your island team";
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/configs/Messages.java:26]: change to
     public String alreadyInYourIsland = "%prefix% &7This player is already in your island .";

File: src/main/java/com/iridium/iridiumskyblock/gui/MembersGUI.java [hunk 3]
```diff
@@ -31,9 +31,9 @@ public MembersGUI(@NotNull Island island) {
     public void onInventoryClick(InventoryClickEvent event) {
         if (members.containsKey(event.getSlot())) {
             User user = members.get(event.getSlot());
-            if (event.getClick().equals(ClickType.RIGHT)) {
+            if (event.getClick().equals(ClickType.LEFT)) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/gui/MembersGUI.java:34]: I agree left seems left seems like it should demote and right should promote, but if your going to change this you also need to change the lore in inventories.json

**Review by PeachesMLG — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/gui/MembersGUI.java (hunk 3 above)
Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/gui/MembersGUI.java:34]: i changed it cuz that already promote = right   demot = left in inventories.(YML)

Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/gui/MembersGUI.java:34]: and should add else if cause users can be click with shift or middle click

**Review by PeachesMLG — APPROVED** [review]

File: src/main/java/com/iridium/iridiumskyblock/configs/Messages.java [hunk 4]
```diff
@@ -23,6 +23,7 @@
     public String invitedPlayer = "%prefix% &7You have invited %player% to join your island";
     public String userInvitedPlayer = "%prefix% &7%inviter% has invited %player% to join your island";
     public String youHaveBeenInvited = "%prefix% &7%inviter% has invited you to join their island";
+    public String alreadyInYourIsland = "%prefix% &7This player is already in your island.";
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/Messages.java:26]: "is already a member of your island" probably would be better

**Review by dlsf — CHANGES_REQUESTED** [review]

**Pull request merged**